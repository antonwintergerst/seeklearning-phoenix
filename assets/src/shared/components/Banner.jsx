import React from 'react';

const Banner = () => (
  <div className="banner">
    <div className="ui container center aligned">
      <h1 className="ui header">Get where you want to be in your career</h1>
    </div>
  </div>
);

export default Banner;
