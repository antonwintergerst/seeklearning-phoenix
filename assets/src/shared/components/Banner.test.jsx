import React from 'react';
import ReactTestRenderer from 'react-test-renderer';
import Banner from './Banner';

describe('Banner', () => {
  it('renders as expected', () => {
    const tree = ReactTestRenderer.create(<Banner />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
