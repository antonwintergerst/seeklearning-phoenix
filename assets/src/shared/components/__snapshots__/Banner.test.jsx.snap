// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Banner renders as expected 1`] = `
<div
  className="banner"
>
  <div
    className="ui container center aligned"
  >
    <h1
      className="ui header"
    >
      Get where you want to be in your career
    </h1>
  </div>
</div>
`;
